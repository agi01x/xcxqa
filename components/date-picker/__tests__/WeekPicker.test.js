import React from 'react';
import { mount } from 'enzyme';
import DatePicker from '..';
import focusTest from '../../../tests/shared/focusTest';

const { WeekPicker } = DatePicker;

describe('WeekPicker', () => {
  focusTest(WeekPicker);

  it('should support style prop', () => {
    const wrapper = mount(
      <WeekPicker style={{ width: 400 }} />
    );
    expect(wrapper.render()).toMatchSnapshot();
  });
});
