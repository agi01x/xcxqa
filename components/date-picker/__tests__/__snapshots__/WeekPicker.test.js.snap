// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekPicker should support style prop 1`] = `
<span
  class="ant-calendar-picker"
  style="width: 400px;"
>
  <span>
    <input
      class="ant-calendar-picker-input ant-input"
      placeholder="Select date"
      readonly=""
      value=""
    />
    <i
      class="anticon anticon-calendar ant-calendar-picker-icon"
    >
      <svg
        aria-hidden="true"
        class=""
        data-icon="calendar"
        fill="currentColor"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
        />
      </svg>
    </i>
  </span>
</span>
`;
